Gems/WhiteBox/Code/CMakeLists.txt

```diff
@@ -198,6 +198,8 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
                     AZ::AzTestShared
                     AZ::AzManipulatorTestFramework.Static
                     Gem::WhiteBox.Editor.Static
+            RUNTIME_DEPENDENCIES
+                Gem::PhysX.Editor
```

Comment:
The UnitTest that runs in the AzTestRunner only loads the dll of the test that is running.

Adding a dependency to `Gem::PhysX.Editor` will not have the `WhiteBox.Editor.Physics.Tests` loading automatically in the AzTestRunner.

The UnitTest code itself would have to load any dependent .dll/.so.

Reply:
Right, the tests here already do that:
https://github.com/o3de/o3de/blob/fed1278fe64e8a435b1c19ba8092b18a98fa9eb1/Gems/WhiteBox/Code/Tests/WhiteBoxPhysicsTest.cpp#L50

Reply:
Alright this is good then.